app/controllers/users_controller.rb

```diff
@@ -5,6 +5,7 @@ class UsersController < ApplicationController
 
   def show
     user = User.find(params[:id])
+    authorize user
```

Comment:
I wouldn't secure this endpoint, unless you want to limit an user's profile only to a signed in user.